res/css/structures/_RoomStatusBar.scss

```diff
@@ -76,6 +76,8 @@ limitations under the License.
         min-height: 70px;
         margin: 12px;
         padding-left: 16px;
+        padding-top: 8px;
+        padding-bottom: 8px;
```

Comment:
Added so the status bar has a little bit more consistent padding when it's squished at smaller widths. It was kinda relying only on the `min-height` before.

The styles here could use some love in general but we can iterate that in a separate PR.